serde/src/de/content.rs

```diff
@@ -0,0 +1,732 @@
+use core::fmt;
```

Comment:
hide the entire module with `![doc(hidden)]` or at least write a doc comment stating that nothing in this module should be used outside of generated code. The PR comments contains all of that already, maybe just place them here?